plugins/polio/js/src/forms/FormAForm.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { FunctionComponent, useEffect, useMemo } from 'react';
+// @ts-ignore
+import { useSafeIntl } from 'bluesquare-components';
+import { Field, useFormikContext } from 'formik';
+import { Grid } from '@material-ui/core';
+import MESSAGES from '../constants/messages';
+import { DateInput } from '../components/Inputs';
+import { useStyles } from '../styles/theme';
+import { MultilineText } from '../components/Inputs/MultilineText';
+import { DebouncedTextInput } from '../components/Inputs/DebouncedTextInput';
+
+type Props = { accessor: string; roundIndex: number };
+
+export const formAFieldNames = [
+    'forma_unusable_vials',
+    'forma_missing_vials',
+    'forma_reception',
+    'forma_usable_vials',
+    'forma_date',
+    'forma_comment',
+];
+
+export const FormAForm: FunctionComponent<Props> = ({
+    accessor,
+    roundIndex,
+}) => {
+    const { formatMessage } = useSafeIntl();
+    const classes: Record<string, string> = useStyles();
+    const {
+        values = {} as any,
+        setFieldTouched,
+        setFieldValue,
+    } = useFormikContext();
+    const fieldValues = useMemo(
+        () => values?.rounds[roundIndex],
+        [roundIndex, values?.rounds],
+    );
+
+    // // Make all fields error by setting touched to true if any other formA field has a value
```

Comment:
i'm not sure of this behaviour

Reply:
The requirement is: the fields can be empty, but if one has a value, then all should have a value (except comments)